lib/modules/manager/github-actions/index.ts

```diff
@@ -9,4 +10,6 @@ export const defaultConfig = {
   ],
 };
 
+export const categories: Category[] = ['ci'];
```

Comment:
Also maybe `cd`? Should we consider combining `cd/cd`?

Reply:
Can definitely argued, but I'm against to combine these. There are managers such as `argocd` which only fit into `cd`.    
Also IMO it's easier for the user to group both together in matching rules them to split them up.